build/.pa11yci.json

```diff
@@ -1,19 +1,24 @@
 {
-  "standard": "WCAG2AA",
+  "standard": "WCAG2AAA",
```

Comment:
Highest level to be as strict as possible.

Reply:
Not sure about this one, as 99.99% of website we are supposed to conform to AA level.
SoI am wondering if seting the desired level to AAA could be damaging rather than helpful, for instance by adding false positive results. 🤔